webapp/ofbiz-tutorial/WEB-INF/controller.xml

```diff
@@ -53,7 +53,24 @@ under the License.
         <security https="true" auth="true"/>
         <response name="success" type="view" value="AddMilerProductFtl"/>
     </request-map>
+    <request-map uri="createUserEvent">
+        <security https="true" auth="true"/>
+        <event type="java" path="org.apache.ofbiz.ofbiztutorial.usermgr.events.UserEvents" invoke="createUserEvent"/>
+        <response name="success" type="view" value="CreateUserFtl" save-home-view="true"/>
```

Comment:
what is save-home-view="true" here

Reply:
By using the save-view-home attribute, you can save your current request view for later access. This feature is handy when you need to retrieve the view-home, representing the last saved home position, in a response. Opting for save-last-view="true" would be more fitting here, as it doesn't refer to the home (main page) position in this context.